librz/main/rizin.c

```diff
@@ -851,12 +851,14 @@ RZ_API int rz_main_rizin(int argc, const char **argv) {
 			return 1;
 		}
 		if (strstr(uri, "://")) {
-			rz_core_cmdf(r, "R+ %s", uri);
+			rz_core_rtr_add(r, uri);
 		} else {
 			argv[opt.ind] = rz_str_newf("http://%s/cmd/", argv[opt.ind]);
-			rz_core_cmdf(r, "R+ %s", argv[opt.ind]);
+			rz_core_rtr_add(r, argv[opt.ind]);
 		}
-		rz_core_cmd0(r, "R!=");
+		// rz_core_cmd0(r, "R!=");
+		RZ_FREE(r->cmdremote);
+		r->cmdremote = rz_str_trim_dup("0");
```

Comment:
Please extract this into `rz_core_rtr_enable(RzCore *, char *cmdremote)` and use it in both cmd_remote.c and here.